src/ext/resources/legacygraphqlapi/schema.graphqls

```diff
@@ -1,3 +1,9 @@
+"""
+Use an asynchronous data fetcher on a separate thread for this field.
+This is only worth it when the execution is long running, i.e. several hundreds of milliseconds, so this doesn't happen by default.
```

Comment:
Is it actually only worth it when the execution takes hundreds of milliseconds? I'm wondering if it would also make sense to run the nearest queries asynchronously?

Reply:
Doesn't the nearest take a few hundred milliseconds? Or at least 100ms?

In any case, I think the nearest query is a good candidate.

Reply:
It probably depends a lot on the parameters, I don't know what is the minimum run time, but it can probably take seconds at worst case. I do agree that it's probably at least worth testing out.

Reply:
One solution (I'm not sure if it's feasible to implement) would be to take the arguments into account while deciding if to execute asynchronously or not. For example, stops query without arguments can be pretty heavy but with arguments not so much.

Reply:
However, I'm perfectly happy with this solution already and I don't think we should spend too much time optimizing this now.

Reply:
Ok, how about this:

- I change the wording so it says "more than 50 milliseconds"
- I add the `@async` directive to the nearest query

Reply:
Sounds good. I will either test tonight or tomorrow how this works in practice and give a final approval.

Reply:
Done